### ISSUE TITLE
1.16.3 loading error - java.lang.ExceptionInInitializerError:null

### ISSUE THREAD

**Issue by baysupercool** [TeamPneumatic/pnc-repressurized#629]
Minecraft Version: 1.16.3

Forge Version: 34.1.15

Mod Version: 1.16.3-2.4.2-49

Simply the game does not load and gives the error in the title

[crash-2020-10-09_17.36.09-fml.txt](https://github.com/TeamPneumatic/pnc-repressurized/files/5355661/crash-2020-10-09_17.36.09-fml.txt)

Edit: No other mods were present.

**Comment by desht** [comment]
Not seeing this behaviour at all.  It loads fine for me with 1.16.3-2.4.2-49 and Forge 34.1.14, either with or without other mods present.

**Comment by desht** [comment]
Also checked with Forge 34.1.15 just in case - still fine.  No crashes.

**Comment by desht** [comment]
```
Exception message: net.minecraft.util.ResourceLocationException: Non [a-z0-9/._-] character in path of location: pneumaticcraft:assembly_program_drıll
```

I'd say your installation is corrupt somehow.  Something's gone and replaced the "i" in "drill" with a "ı" character - https://www.compart.com/en/unicode/U+0131.  No idea what's done that or why, but it certainly wasn't me.

**Comment by desht** [comment]
OK, think I know what's happening.  What language are you running with? Turkish maybe?  There's a translation issue when certain strings are lowercased within the mod, which should be an easy fix. Will get a new build out in the next day or so.  In the meantime a workaround should be to run with an en_US locale.

**Comment by baysupercool** [comment]
Yes actually my computer language is Turkish.
I have experienced a crash like this before on a different mod. It capitalized "stair" wrong. Something similar I think.

**Comment by desht** [comment]
This will be fixed in the next release (2.4.3) ~~but you can use build 50 from https://jenkins.k-4u.nl/job/PneumaticCraft-Repressurized-1.16/ if you'd like to test the fix now...~~  sorry build server problems right now, will hopefully have a build up later.

**Comment by desht** [comment]
OK, build 54 from https://jenkins.k-4u.nl/job/PneumaticCraft-Repressurized-1.16/ should work.

**Comment by baysupercool** [comment]
Build 54 works, no crashes.

**Comment by desht** [comment]
Fixed in 2.4.3 release
